tests/test_scripts.py

```diff
@@ -82,3 +84,75 @@ def test_proto_conversion(tmpdir):
     assert p.returncode == 0
 
     assert p.stdout.startswith(b'{\n  "meta": ') or p.stdout.startswith(b'{\r\n  "meta": ')
+
+
+def run_detect_duplicate_features(rule_dir, rule_path):
+    # rule_path = "collection/credit-card/parse-credit-card-information.yml"
+    args = [rule_dir, rule_path]
+    script_path = get_script_path("detect_duplicate_features.py")
+    args = [sys.executable] + [script_path] + args
+    print(f"running: '{args}'")
+    return subprocess.run(args)
+
+
+def test_detect_duplicate_features(tmpdir):
+    RULESET = {
+        "rule_1": textwrap.dedent(
+            """
+                rule:
+                    meta:
+                        name: Test Rule 1
+                        scope: function
+                    features:
+                      - or:
```

Comment:
it would be good to have at least one rule with one or a few extra levels of depth to also show the child features are checked